index.html

```diff
@@ -5153,6 +5165,73 @@ <h4><dfn>Is Element Enabled</dfn></h4>
  <li><p>Return <a>success</a> with data <var>enabled</var>.
 </ol>
 </section> <!-- /Is Element Enabled -->
+
+<section>
+  <h4><dfn>Get Computed Role</dfn></h4>
+
+  <table class="simple jsoncommand">
+    <tr>
+      <th>HTTP Method</th>
+      <th>URI Template</th>
+    </tr>
+    <tr>
+      <td>GET</td>
+      <td>/session/{<var>session id</var>}/element/{<var>element id</var>}/computedrole</td>
+    </tr>
+  </table>
+
+  <p>The <a>remote end steps</a> are:
+
+  <ol>
+    <li><p>If the <a>current browsing context</a> is <a>no longer open</a>,
+        return <a>error</a> with <a>error code</a> <a>no such window</a>.
+
+    <li><p><a>Handle any user prompts</a>
+        and return its value if it is an <a>error</a>.
+
+    <li><p>Let <var>element</var> be the result
+        of <a>trying</a> to <a>get a known connected element</a>
+        with <a>url variable</a> <var>element id</var>.
+
+    <li><p>Let <var>role</var> be the result of computing the <a>WAI-ARIA role</a>.
```

Comment:
Presumably this is `of <var>element</var>`.

Reply:
fixed